src/query/management/src/role/role_mgr.rs

```diff
@@ -256,32 +258,43 @@ impl RoleApi for RoleMgr {
     #[minitrace::trace]
     async fn transfer_ownership_to_admin(
         &self,
-        object: &OwnershipObject,
+        objects: &Vec<OwnershipObject>,
     ) -> databend_common_exception::Result<()> {
         // Ensure accurate matching of a key
-        let match_seq = MatchSeq::GE(1);
-        let key = self.ownership_object_key(object);
+        let mut if_then = vec![];
+        for object in objects {
+            let owner_key = self.ownership_object_key(object);
+            let owner_value = serialize_struct(
+                &OwnershipInfo {
+                    object: object.clone(),
+                    role: "account_admin".to_string(),
+                },
+                ErrorCode::IllegalUserInfoFormat,
+                || "",
+            )?;
+            if_then.push(txn_op_put(&owner_key, owner_value))
```

Comment:
Do not you need to add `txn_cond(&key, GE, 1)` for every owner_key?

In the previous version of this PR, you were updating with `MatchSeq::GE(1)`.

Reply:
My initial appeal was just to change the owner of the object owned by the deleted role to object. Now that these objects are fetched in the outer for loop, I think maybe we don't need to match the seq?

Reply:
You always need to match the seq otherwise you can not be sure you are removing the key you want to remove.

Reply:
Ok I got you. That means in meta kv the seq is the real key?

Reply:
Yes, `(key, seq)` is considered the globally unique key for a record